common/src/main/java/com/wynntils/features/user/QuickCastFeature.java

```diff
@@ -4,18 +4,20 @@
  */
 package com.wynntils.features.user;
 
+import static com.wynntils.wynn.objects.SpellDirection.NO_SPELL;
```

Comment:
Please do not do static imports, instead quality `NO_SPELL` to `SpellDirection.NO_SPELL`.